php/libraries/NDB_BVL_InstrumentStatus.class.inc

```diff
@@ -236,12 +236,12 @@ class NDB_BVL_InstrumentStatus
     /**
      * Sets the current administration status of the instrument
      *
-     * @param string $status the new status from the set
-     *                       (NULL, 'None','Partial','All')
+     * @param ?string $status the new status from the set
+     *                        (NULL, 'None','Partial','All')
      *
      * @return void
      */
-    function setAdministrationStatus(string $status): void
+    function setAdministrationStatus(?string $status): void
```

Comment:
As stated in the description of the function, `$status` should be allowed to be `null`. These changes are necessary to accept null as a valid status.